### PR TITLE
MBL-1318: Spiking crash PicassoDrawable.draw trying to draw too large bitmap mitigation

### PR DESCRIPTION
**Pull request by Arkariang**
# 📲 What

- Avoid usage of third parties on VM's use ImageView extension instad.
- Deprecate as much use of Picasso as possible.
- Every ImageViewExtension method is covered now by try/catch blocks and will send information to firebase in case of error containing the URL

# 🤔 Why

- Picasso is not very performance library, the recent crashes related to images has shown `Glide` could have minimized the effect:

We discovered debuging alongside #infra,- the actual fix has to been done on by infra-, but there are some improvements that can mitigate it on our side: 

Basically the library used right now (Picasso), is way less performant that others, during our debug session we were able to identify a crashing images with Picasso, but when using other library (Glide) did not crash, as is way more performance. 

# 🛠 How
- Identified all usage of Picasso, those who where not using ImageView Extension were changed
- `ImageViewExt.kt` now holds all occurencies of `Glide`, exchange the image library in the future will be WAY easier this way.

# 👀 See
- No user facing, but make sure images load everywhere around the app

|  |  |

# 📋 QA

- Load projects, backed projects, Project stories with gifts ... as much as you can.

# Story 📖

[MBL-1318](https://kickstarter.atlassian.net/browse/MBL-1318)

[MBL-1318]: https://kickstarter.atlassian.net/browse/MBL-1318?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by leighdouglas — APPROVED** [review]
🎉

**Review by mtgriego — APPROVED** [review]

**Review by ycheng-kickstarter — APPROVED** [review]
LGTM

File: app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt [hunk 1]
```diff
@@ -3,75 +3,162 @@ package com.kickstarter.ui.extensions
...
+//                            override fun onSuccess() {
+//                                imageViewPlaceholder?.setImageDrawable(target.drawable)
+//                            }
+//
+//                            override fun onError(e: Exception?) {
+//                                target.setImageDrawable(null)
+//                                imageViewPlaceholder?.setImageDrawable(null)
+//                            }
+//                        }
+//                    )
```
Comment (ycheng-kickstarter) [inline app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt:103]: Did you mean to leave these comments in?

Comment (Arkariang) [reply app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt]: nop!

**Review by ycheng-kickstarter — APPROVED** [review]

**Pull request merged**